components/resourceDetails/resourceSharing/agentAccessTable/index.jsx

```diff
@@ -234,15 +193,14 @@ export default function AgentAccessTable({ type }) {
                   );
                 })
               ) : (
-                <span className={classes.emptyStateTextContainer}>
-                  <p>
-                    {selectedTabValue === ""
-                      ? emptyStateText
-                      : `No ${
-                          selectedTabValue === "Person" ? "people " : "groups "
-                        } found`}
-                  </p>
-                </span>
+                <tr className={classes.emptyStateTextContainer}>
+                  <td>
+                    {selectedTabValue &&
+                      `No ${
+                        selectedTabValue === "Person" ? "people " : "groups "
```

Comment:
This sounds like something that should be assigned to the type objects? E.g. what happens when we introduce `Applications` into this mix.

Reply:
This table is to display permissions, not contacts, and since we haven't dealt with Groups yet (there's a ticket for that) in the permissions, the tabs are still just using strings (as opposed to the multiselect agent picker that uses the Group and Person objects).